lib/api/2.0/nodes.js

```diff
@@ -78,9 +77,10 @@ var nodesGetWorkflowById = controller(function(req) {
         newQuery = req.query;
     }
 
-    return workflowApiService.getWorkflowsByNodeId(req.swagger.params.identifier.value,
-                                                   newQuery);
-
+    return waterline.nodes.needByIdentifier(req.swagger.params.identifier.value)
+    .then(function() {
+        return nodes.getNodeWorkflowById(req.swagger.params.identifier.value, newQuery);
```

Comment:
`nodes/getNodeworkflowById` (https://github.com/RackHD/on-http/blob/master/lib/services/nodes-api-service.js#L541) doesn't support the 2nd argument `newQuery`, I believe this is the reason of jenkins failure.